wsmaster/che-core-api-machine-shared/src/main/java/org/eclipse/che/api/machine/shared/dto/execagent/event/DtoWithPidDto.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.machine.shared.dto.execagent.event;
+
+/**
+ * @author Dmitry Kuleshov
+ */
+public interface DtoWithPidDto {
```

Comment:
The first 'DTO' is corresponding to the nature of an interface (which is DTO with process identifier), while the second DTO is used in accordance to our DTO naming conventions. I don't think that `HasPidDto` or `ProcessDto` would more accurately express the idea that stands behind this interface. Probably something like `EventWithPidDto` could be used but that is also arguable.